### PR TITLE
Mongo insert

### PR DESCRIPTION
**Pull request by buckley-w-david**
Load csv data into mongodb

### CODE REVIEW

**Comment by buckley-w-david** [comment]
Apologies for the length of PR, the vast majority is test data

File: python/src/energuide/database.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    collection = 'ENERGUIDE_COLLECTION'
+
+
+class EnvDefaults(enum.Enum):
+    username = ''
+    password = ''
+    host = 'localhost'
+    port = 27017
+    database = 'energuide'
+    collection = 'buildings'
```
Comment (JasonMWhite) [inline python/src/energuide/database.py]: let's call it `dwellings` for now

File: python/src/energuide/database.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        collection = database[collection_name]
+
+        chunks = pd.read_csv(csvfile, chunksize=CHUNKSIZE)
+
+        for dataframe in chunks:
+            if columns is None:
+                columns = dataframe.columns
+            dataframe = dataframe.where((pd.notnull(dataframe)), None)
+
+            inserts = map(pymongo.InsertOne, dataframe[columns].to_dict('records'))
```
Comment (JasonMWhite) [inline python/src/energuide/database.py]: I don't know why this didn't fail the linter - using map this way is discouraged in Python3. A for loop is clearer and preferable.

File: python/tests/conftest.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+import pymongo
+import pytest
+from energuide import database
+
+@pytest.fixture
+def username() -> str:
+    return os.environ.get(database.EnvVariables.username.value, database.EnvDefaults.username.value)
+
+@pytest.fixture
```
Comment (JasonMWhite) [inline python/tests/conftest.py]: There's also a PEP rule about 2 lines between module-level functions/classes. It's not in Pylint for some reason, but it drives my editor nuts.

**Comment by JasonMWhite** [comment]
I think you meant to remove `test_cli.py`

**Pull request merged**